csv-to-db/docker-compose.yml

```
@@ -9,3 +9,6 @@ services:
       - POSTGRES_PASSWORD=kaogeek
     ports:
       - "5432:5432"
+    volumes:
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
```

I think this solution is cleaner than my script that manually drop existing and create a new DB. It might be good if you can rebase on top of my latest changes and apply this change to it.

---

Rebased.

I am not sure which change I need to apply. Then I comes with questions.
1. From my guess, does it mean that I need to remove your previous solution (like `import_data` and `run_sql` shell scripts)?
2. This PR aims improvement of how we insert data to Postgres (by using `COPY` command). Would it be good whether other thing else could be fulfilled in your PR instead? (by lemme review your PR) :)

---

I was thinking of just:

```suggestion
      - ./create_report_table.sql:/docker-entrypoint-initdb.d/init.sql
```

Does this make sense?